test/functional/page_objects/common_page.ts

```diff
@@ -101,8 +101,10 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
           `Found login page.  Logging in with username = ${config.get('servers.kibana.username')}`
         );
         await PageObjects.shield.login(
-          config.get('servers.kibana.username'),
-          config.get('servers.kibana.password')
+          // config.get('servers.kibana.username'),
+          // config.get('servers.kibana.password')
+          'test_user',
+          'changeme'
```

Comment:
I think we need to add test user straight to the config and read it similar way, we probably want to rename existing one to `defaultUser` and may this `testUser` for this one?
```
          config.get('servers.kibana.testUser.username'),
          config.get('servers.kibana.testUser.password')
```